bin/publish

```diff
@@ -7,7 +7,7 @@ REDHAT_IMAGE='scan.connect.redhat.com/ospid-1c46a2de-1d88-40e6-a433-7114ad0099cb
 readonly REGISTRY="cyberark"
 readonly VERSION="$(short_version_tag)"
 
-readonly IMAGES=(
+readonly DESTINATION_IMAGES=(
   "conjur-kubernetes-authenticator"
```

Comment:
are these two images the same? just named differently? If so why do we have this duplicate?

Reply:
we changed the name from `conjur-kubernetes-authenticator` to `conjur-authn-k8s-client` but we still push the first one for backwards-compatibility. I think it is deprecated so it won't stay with us for much longer.

Reply:
may be worth commenting that are clarity?

Reply:
Done